GoldRush.js

```diff
@@ -0,0 +1,93 @@
+// const Matrix = require("./Matrix");
+
+class GoldRush extends Matrix {
+    constructor(rowNum, colNum) {
+        super(rowNum, colNum)
+        this.generateMatrix(rowNum, colNum)
+        this.addCoins(rowNum, colNum)
+        this.alter(0, 0, 1)
+        this.alter(4, 4, 2)
+        this.playerScores = {
+            1: 0,
+            2: 0
+        }
+    }
+
+
+    movePlayer(playerNum, direction) {
+        let x = this.findCoordinate(playerNum).y
```

Comment:
Don't find coordinate twice. This is a lot of `for` loops. Instead, find the coordinate once and then use `x` and `y` like so:
`const coord = this.findCoordinate(playerNum)
const x = coord.x` etc...

You can also use destructuring to simplify this:
`const { x, y } = this.findCoordinate(playerNum)`

In any case, you should be using `const` and not `let`.